### ISSUE TITLE
Feature request: inodes please

### ISSUE THREAD

**Issue by miltieIV2** [jmcgrath207/k8s-ephemeral-storage-metrics#128]
We recently deployed this, only to find that there are no metrics for inodes, which is what we ran out of.

**Comment by jmcgrath207** [comment]
Hey @miltieIV2,

Ah, sorry to hear that. Were you only interested in the `inodesUsed` metric?

FWIW Here is the full raw query from the proxied `cadvisor` 
```
kubectl get --raw "/api/v1/nodes/minikube/proxy/stats/summary" | jq '.pods |  to_entries | map(select(.value.podRef.name| match("shrink-test-7d6fbd9dcc-crvl9")))'
...
      "volume": [
        {
          "time": "2024-11-21T21:58:07Z",
          "availableBytes": 81934872576,
          "capacityBytes": 947380916224,
          "usedBytes": 2043904,
          "inodesFree": 46797174,
          "inodes": 58818560,
          "inodesUsed": 484,
          "name": "cache-volume-3"
        },
        {
          "time": "2024-11-21T21:58:07Z",
          "availableBytes": 81934872576,
          "capacityBytes": 947380916224,
          "usedBytes": 2080768,
          "inodesFree": 46797174,
          "inodes": 58818560,
          "inodesUsed": 491,
          "name": "cache-volume-1"
        },
        {
          "time": "2024-11-21T21:58:07Z",
          "availableBytes": 32926322688,
          "capacityBytes": 32926334976,
          "usedBytes": 12288,
          "inodesFree": 4019319,
          "inodes": 4019328,
          "inodesUsed": 9,
          "name": "kube-api-access-zfklf"
        }
      ],
      "ephemeral-storage": {
        "time": "2024-11-21T21:58:42Z",
        "availableBytes": 81935126528,
        "capacityBytes": 947380916224,
        "usedBytes": 6160960,
        "inodesFree": 46797267,
        "inodes": 58818560,
        "inodesUsed": 978
      },

```

**Comment by miltieIV2** [comment]
I'm not seeing the `ephemeral-storage` metric from my k8s v.1.25 kubelet:10250 /metrics or /metrics/cadvisor

**Comment by jmcgrath207** [comment]
Try this.

`kubectl get --raw "/api/v1/nodes/{your_node}/proxy/stats/summary" `

**Comment by miltieIV2** [comment]
ok, yeah, that's working great.
Now where do I find that way down on the other end in Prometheus.

**Comment by miltieIV2** [comment]
Anyway, we'd like to see inodes added to the these storage metrics.

**Comment by jmcgrath207** [comment]
@miltieIV2 
This feature is now released in version `1.16.0`.

@mattcontinisio Thank you for contributing to adding this feature.
